### PR TITLE
Adjust Button Styling to Match Intended UI from Content

### PR DESCRIPTION
**Pull request by mark-fitzgerald**
## Summary:
Adjust button styling to be left-aligned, and to use the line spacing of the surrounding text.

Issue: none

## Before:
Screenshot 2024-04-08 at 1 09 13 PM

## After:
Screenshot 2024-04-08 at 1 08 12 PM

### CODE REVIEW

**Review by nishasy — APPROVED** [review]
Nice!

File: packages/perseus/src/widgets/explanation.tsx [hunk 1]
```diff
@@ -92,14 +92,16 @@ class Explanation extends React.Component<Props, State> {
         // While the button is not normally included in a block of text, it needs to be able to accommodate such a case.
         const buttonStyleOverrides = {
             height: "auto",
+            lineHeight: "inherit",
```
Comment (jeremywiebe) [inline packages/perseus/src/widgets/explanation.tsx:95]: Not directly related to this change, but these styles (`buttonStyleOverrides` and `labelStyle`) could (should?) go into the `const styles` object later on if there's no dynamic style calculation in them.

**Review by jeremywiebe — APPROVED** [review]
One comment (non-blocking)

File: packages/perseus/src/widgets/explanation.tsx (hunk 1 above)
Comment (mark-fitzgerald) [reply packages/perseus/src/widgets/explanation.tsx:95]: (face-palm)
I over-read the WB documentation and implied something that wasn't there. Still getting used to this Stylesheet (CSS-in-JS) implementation.

**Pull request merged**